src/include/ipc/trace.h

```diff
@@ -51,6 +51,19 @@ struct sof_ipc_dma_trace_posn {
 	uint32_t messages;	/* total trace messages */
 } __attribute__((packed));
 
+/* part of sof_ipc_dma_trace_filter */
+struct sof_ipc_dma_trace_filter_elem {
+	uint32_t uuid_addr;	/* component type identification */
+	int32_t lvl;		/* log level */
+} __attribute__((packed));
```

Comment:
Just log_level here. lvl is pretty hard to read.

Reply:
changed